src/components/inputs/IconCheckbox/IconCheckbox.scss

```diff
@@ -0,0 +1,27 @@
+
+.IconCheckbox {
+	height: 24px;
+	width: 24px;
+	position: relative;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+
+	&:hover {
+		background-color: #f7f6f6;
```

Comment:
Would you tweak this to use the sass variable `$gray5`?

Reply:
Now wondering if this is the value for both light and dark modes 🤔

Reply:
If so, perhaps you can reuse one of the mixins for that color background: 
![image](https://user-images.githubusercontent.com/41925404/93352586-9d3e0f80-f800-11ea-9ab4-e7a6a9e00d57.png)